main.go

```diff
@@ -35,18 +35,18 @@ func (s ParcelService) Register(client int, address string) (Parcel, error) {
 		Client:    client,
 		Status:    ParcelStatusRegistered,
 		Address:   address,
-		CreatedAt: time.Now().UTC().Format(time.RFC3339),
+		Created_at: time.Now().UTC().Format(time.RFC3339),
```

Comment:
В Го как раз принято использование `PascalCase`
```suggestion
		CreatedAt: time.Now().UTC().Format(time.RFC3339),
```

Reply:
я понимаю, что нужно использовать CamelCase, однако в бд именно так называется поле (как в snake_case). Менять бд я не хотела бы,  вдруг вообще все перестанет работать..